schema/tables.sql

```diff
@@ -2081,11 +2143,22 @@ CREATE TABLE `DiffractionPlan` (
   `monoBandwidth` double DEFAULT NULL,
   `centringMethod` enum('xray','loop','diffraction','optical') DEFAULT NULL,
   `userPath` varchar(100) DEFAULT NULL COMMENT 'User-specified relative "root" path inside the session directory to be used for holding collected data',
+  `columnId` int(10) unsigned DEFAULT NULL,
+  `experimentTypeId` int(10) unsigned DEFAULT NULL,
+  `robotPlateTemperature` float DEFAULT NULL COMMENT 'units: ',
+  `exposureTemperature` float DEFAULT NULL COMMENT 'units: ',
+  `qMin` float DEFAULT NULL COMMENT 'minimum in qRange',
```

Comment:
I think qMin and qMax can be essentially represented as resolution. What units do they want to specify these in? use minres, maxres instead?

Reply:
I don't know ... It's in the requirements for i22, page 6 in the requirements doc. I'll try to find out.